frontend_app_file/src/container/File.jsx

```diff
@@ -932,7 +932,7 @@ class File extends React.Component {
                   fileType={state.content.file_extension}
                   fileSize={displayFileSize(state.content.size)}
                   filePageNb={state.content.page_nb}
-                  creationDate={displayDistanceDate(state.content.created, state.loggedUser.lang)}
+                  creationDate={(new Date(state.content.created)).toLocaleString(props.i18n.language, { day: '2-digit', month: '2-digit', year: 'numeric' })}
```

Comment:
to keep consistency with other props, rename this props `creationDateFormattedWithTime`